src/Components/Shared/Inputs/inputs.module.css

```diff
@@ -32,7 +32,8 @@
 }
 
 .error {
-  color: #ff0000;
-  font-size: 20px;
+  color: rgb(255, 255, 255);
```

Comment:
Error color shouldn't be white, white doesn't express error, this should be a red tone